### PR TITLE
Remove example of matrix of size (12, 12) and (64, 64)

### PR DESCRIPTION
**Pull request by patniharshit**
refs #7956 Only keep middle plots
Remove first and third example and keep second one.

### CODE REVIEW

File: examples/pylab_examples/matshow.py [hunk 1]
```diff
@@ -10,17 +10,7 @@ def samplemat(dims):
         aa[i, i] = i
```
Comment (anntzer) [inline examples/pylab_examples/matshow.py]: Could be rewritten as `np.fill_diagonal(aa, range(min(dims)))`.

Comment (dstansby) [inline examples/pylab_examples/matshow.py]: I think the current version is much more readable, which is important for an example.

Comment (anntzer) [inline examples/pylab_examples/matshow.py]: I'm fine either way.

File: examples/pylab_examples/matshow.py [hunk 2]
```diff
@@ -10,17 +10,7 @@ def samplemat(dims):
...
-    plt.matshow(samplemat(d))
-
-# Fixing random state for reproducibility
-np.random.seed(19680801)
-
-
-# Display a random matrix with a specified figure number and a grayscale
-# colormap
-plt.matshow(np.random.rand(64, 64), fignum=100, cmap=plt.cm.gray)
+# Display  matrix
```
Comment (anntzer) [inline examples/pylab_examples/matshow.py]: two spaces, and trailing whitespace.

Comment (patniharshit) [inline examples/pylab_examples/matshow.py]: Should I remove spaces ?

Comment (anntzer) [inline examples/pylab_examples/matshow.py]: one before "matrix" and whatever is after.

**Comment by tacaswell** [comment]
@anntzer I find your review confusing, you asked for changes with the comment 'LGTM' ;)

**Comment by anntzer** [comment]
I noticed the spaces issue afterwards and forgot to edit the main message :-)

**Comment by dstansby** [comment]
Backported to 2.0.x via https://github.com/matplotlib/matplotlib/commit/35996ce63d5865f7a101d828a3f6efb483013b46

**Pull request merged**